### ISSUE TITLE
最新的fast_inference_报错

### ISSUE THREAD

**Issue by tobeornottoer** [RVC-Boss/GPT-SoVITS#1487]
rguments for call are not valid.
The following variants are available:
  
  aten::eq.Tensor(Tensor self, Tensor other) -> Tensor:
  Expected a value of type 'Tensor' for argument 'other' but instead found type 'bool'.
  
  aten::eq.Scalar(Tensor self, Scalar other) -> Tensor:
  Expected a value of type 'number' for argument 'other' but instead found type 'bool'.
  
  aten::eq.Scalar_out(Tensor self, Scalar other, *, Tensor(a!) out) -> Tensor(a!):
  Expected a value of type 'number' for argument 'other' but instead found type 'bool'.
  
  aten::eq.Tensor_out(Tensor self, Tensor other, *, Tensor(a!) out) -> Tensor(a!):
  Expected a value of type 'Tensor' for argument 'other' but instead found type 'bool'.
  
  aten::eq.int_list(int[] a, int[] b) -> bool:
  Expected a value of type 'List[int]' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.device(Device a, Device b) -> bool:
  Expected a value of type 'Device' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.bool(bool a, bool b) -> bool:
  Expected a value of type 'bool' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.enum(AnyEnumType a, AnyEnumType b) -> bool:
  Expected a value of type 'AnyEnumType' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.int(int a, int b) -> bool:
  Expected a value of type 'int' for argument 'b' but instead found type 'bool'.
  
  aten::eq.complex(complex a, complex b) -> bool:
  Expected a value of type 'complex' for argument 'b' but instead found type 'bool'.
  
  aten::eq.float(float a, float b) -> bool:
  Expected a value of type 'float' for argument 'b' but instead found type 'bool'.
  
  aten::eq.int_float(int a, float b) -> bool:
  Expected a value of type 'float' for argument 'b' but instead found type 'bool'.
  
  aten::eq.float_int(float a, int b) -> bool:
  Expected a value of type 'int' for argument 'b' but instead found type 'bool'.
  
  aten::eq.float_complex(float a, complex b) -> bool:
  Expected a value of type 'complex' for argument 'b' but instead found type 'bool'.
  
  aten::eq.complex_float(complex a, float b) -> bool:
  Expected a value of type 'float' for argument 'b' but instead found type 'bool'.
  
  aten::eq(Scalar a, Scalar b) -> bool:
  Expected a value of type 'number' for argument 'b' but instead found type 'bool'.
  
  aten::eq.str(str a, str b) -> bool:
  Expected a value of type 'str' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.float_list(float[] a, float[] b) -> bool:
  Expected a value of type 'List[float]' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.Tensor_list(Tensor[] a, Tensor[] b) -> bool:
  Expected a value of type 'List[Tensor]' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.bool_list(bool[] a, bool[] b) -> bool:
  Expected a value of type 'List[bool]' for argument 'a' but instead found type 'Tensor'.
  
  aten::eq.str_list(str[] a, str[] b) -> bool:
  Expected a value of type 'List[str]' for argument 'a' but instead found type 'Tensor'.
  
  eq(float a, Tensor b) -> Tensor:
  Expected a value of type 'Tensor' for argument 'b' but instead found type 'bool'.
  
  eq(int a, Tensor b) -> Tensor:
  Expected a value of type 'Tensor' for argument 'b' but instead found type 'bool'.
  
  eq(complex a, Tensor b) -> Tensor:
  Expected a value of type 'Tensor' for argument 'b' but instead found type 'bool'.

The original call is:
  File "/home/trumanwong/GPT-SoVITS/GPT_SoVITS/AR/models/t2s_model.py", line 153
            for i in range(batch_size):
                # mask = padding_mask[i,:,0]
                idx = torch.where(padding_mask[i,:,0]==False)[0]
                                  ~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
                x_item = x[i,idx,:].unsqueeze(0)
                attn_item = attn[i,idx,:].unsqueeze(0)

**Comment by tobeornottoer** [comment]
之前是可以的，就更新成最新的就跑不起来了

**Comment by ChasonJiang** [comment]
能否给出相关的推理参数？

**Comment by ChasonJiang** [comment]
另外，pytorch版本是啥？

**Comment by tobeornottoer** [comment]
我就直接从webui上面打开推理UI就报这个错
python库的版本：
pytorch-lightning                        2.3.3
pytorch-wpe                                0.0.1
rotary-embedding-torch             0.6.4
torch                                            2.0.1+cu118
torch-complex                            0.4.4
torchaudio                                  2.0.2+cu118
torchmetrics                               1.4.0.post0

**Comment by tobeornottoer** [comment]
就是直接打开推理的webui
python GPT_SoVITS/inference_webui.py

**Comment by ChasonJiang** [comment]
pytorch版本的问题应该是，我的版本太高了，每个版本的torchscript的要求不同。如果你急着用可以注释掉GPT_SoVITS\AR\models\t2s_model.py中所有的“@torch.jit.script”

**Comment by tobeornottoer** [comment]
好，谢谢大佬
